src/gui/rss/rsswidget.cpp

```diff
@@ -455,13 +455,17 @@ void RSSWidget::handleCurrentArticleItemChanged(QListWidgetItem *currentItem, QL
     auto article = m_articleListWidget->getRSSArticle(currentItem);
     Q_ASSERT(article);
 
+    const QColor highlightedBase = m_ui->textBrowser->palette().color(QPalette::Highlight);
```

Comment:
I would add `Color` suffix to these variable names. Also I would store them as QString (since you never use any QColor properties except the name).

Reply:
adding Color suffix makes variable name too big, but anyway I'll change these types to QString and add the suffix too.

Reply:
Too big name is anyway better then confusing one...